plugins/extensions/plugins/DeflectPlugin.cpp

```diff
@@ -187,12 +206,9 @@ void DeflectPlugin::_send(
 {
     deflect::ImageWrapper deflectImage( imageData, windowSize.x(), windowSize.y(), deflect::RGBA );
 
-    ApplicationParameters& applicationParameters =
-        _brayns.getParametersManager().getApplicationParameters();
-
-    deflectImage.compressionQuality = applicationParameters.getJpegCompression();
+    deflectImage.compressionQuality = _params.getQuality();
     deflectImage.compressionPolicy =
-        ( deflectImage.compressionQuality != 100 ) ?
+        _params.getCompression() ?
```

Comment:
Following our discussion, it's probably better to have 2 different parameters for the quality and the activation of the compression. For the records, compressing to 100% gives the same quality as compression disabled, but removes the Alpha information.